src/Components/Event/Event.jsx

```diff
@@ -0,0 +1,138 @@
+                  <h3>DATE</h3>
+                  <p>Stage 1 :{item.stage1_datetime}</p>
+                  <p>{item.stage1_venue}</p>
+                  <br />
+                  <p>Stage 2 :{item.stage2_datetime}</p>
+                  <p>{item.stage2_venue}</p>
+                </div>
+              );
+            })}
+          </div>
+        </div>
+        <div className={styles.content}>
+          <h1>Rules and Regulations</h1>
+          <div className={styles.overview}>
+            <h2 className={styles.head1}>OVERVIEW</h2>
+            <br />
+            <p>
+              {data.content.overview.map((val) => {
+                return (
+                  <>
+                    <p>{val}</p>
+                    <br />
+                  </>
+                );
+              })}
+            </p>
+            <br />
+            <p>
+              {data.content.outcome.map((val) => {
+                return (
+                  <>
+                    <p>{val}</p>
+                    <br />
+                  </>
+                );
+              })}
+            </p>
+            <br />
+          </div>
+          <div className={styles.stage1}>
+            {data.content.stage1.map((item) => {
+              return (
+                <div className={styles.overview} key={item.stage_id}>
+                  <h2 className={styles.head1}>{item.stage_name}</h2>
+                  <br />
+                  <p>
+                    {item.stage_rules.map((val) => {
+                      return (
+                        <>
+                          <p>{val}</p>
+                          <br />
+                        </>
+                      );
+                    })}
+                  </p>
+                  <br />
+                  <a href={item.decklink} target="_blank">
+                    Click here.
+                  </a>
+                </div>
+              );
+            })}
+          </div>
+          <div className={styles.stage2}>
+            {data.content.stage2.map((item) => {
+              return (
+                <div className={styles.overview} key={item.stage_id}>
+                  <h2 className={styles.head1}>{item.stage_name}</h2>
+                  <br />
+                  <p>
+                    {item.stage_rules.map((val) => {
+                      return (
+                        <>
+                          <p>{val}</p>
+                          <br />
+                        </>
+                      );
+                    })}
+                  </p>
+                </div>
+              );
+            })}
+          </div>
+          <div className={styles.overview}>
+            <h2 className={styles.head1}>GUIDELINES</h2>
+            <br />
+            <p>
+              {data.content.guidelines.map((val) => {
+                return (
+                  <>
+                    <p>{val}</p>
+                    <br />
+                  </>
```

Comment:
every `map` should have a key prop while rendering list of items in react. 

return a `div` with `p` and `br` as child with key prop in div